pack/debian/changelog

```diff
@@ -1,6 +1,12 @@
-srain (1.0.0) stable; urgency=medium
+srain (1.0.1-1) stable; urgency=medium
```

Comment:
I think it should be "unstable". It was my mistake. I would glad if you can correct this. :smile:

Reply:
fixed, thanks